eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventSenderOptions.java

```diff
@@ -58,4 +74,36 @@ public String partitionId() {
         return partitionId;
     }
 
+    /**
+     * Gets the default timeout when sending events.
+     *
+     * @return The default timeout when sending events.
+     */
+    public Duration timeout() {
+        return timeout;
+    }
+
+    /**
+     * Creates a shallow clone of this instance.
+     *
+     * The object is cloned, but the objects {@link #retry()}, {@link #timeout()} and {@link #partitionId()} are not
+     * cloned. {@link Duration} and {@link String} are immutable objects and are not an issue. However, the
+     * implementation of {@link Retry} could be mutable.
+     *
+     * @return A shallow clone of this object.
+     */
+    public EventSenderOptions clone() {
+        EventSenderOptions clone;
+        try {
+            clone = (EventSenderOptions) super.clone();
```

Comment:
It's a bird!  It's a plane!  It's.... Super Clone!

![](https://media2.giphy.com/media/SBDwBgzOoclxe/giphy.gif)

Reply:
Ahaha omg. That made my day. 🤣